.buildkite/pipeline.yml

```diff
@@ -293,7 +293,7 @@ steps:
       queue: central
       slurm_ntasks: 1
 
-  - label: ":computer: Advection CG plane limiter cosine bells"
+  - label: ":computer: Plane limiters advection cosine bells"
```

Comment:
I don't have a problem with renaming these, but I would prefer it to be a different PR

Reply:
Ok, sure. I opened #700 and dropped the commit here.